js/web/components/HomePage.js

```diff
@@ -65,33 +68,43 @@ const HomePage = () => {
           align="left"
           sx={{ paddingBottom: { md: "140px", xs: "30px" } }}
         >
-          When an artist publishes on Nina, their tracks are permanently hosted
-          for both streaming and purchasing. Anyone can listen. By purchasing a
-          release, fans support the artist directly - Nina does not take a cut.
-          Soon, we will provide artists with the tools to engage these
-          supporters.
+          Music on Nina can be publicly streamed by anyone, while also being released in the form of a digital edition as scarce or ubiquitous as the artist desires. You can use Nina to simply host your music, to sell digital editions, or to build out a patronage mechanism by providing unique content + experiences to paying supporters.
+          <br /><br />More questions? Read our <Link href="/faq">FAQ</Link>.
```

Comment:
elements that need `<br>` should be their own typography component, u can pass them a `mt` prop which will apply marginTop without having to hit sx or styledComponents